### ISSUE TITLE
Clarity for link-to active class

### ISSUE THREAD

**Issue by j-brennan** [balinterdi/rarwe-issues#274]
File version: `Rock and Roll with Ember.js 3 (20180302).pdf`

In Chapter 8 on page 131 you mention that the appropriate band link in the side bar is no longer marked as active. You explain this by:

```
This is because bands.band.index is not a parent route of either
bands.band.songs or bands.band.details and thus is not marked
as active when we're on either of those pages.
```

This is the first time you have talked about `active`. It might be easier to understand for new Ember readers if you explain how the `link-to` component adds a class of `active` when the rendered link matches the current route.

The `active` class is first introduced in Chapter 4 on page 36 - in a blob of CSS. It is not until later in Chapter 4 on page 46 under the section `Showing a list of bands` that we see a screenshot showing the Bands link with a yellow underline. Maybe here would be a good place to describe why the Bands link has the yellow underline?

**Comment by balinterdi** [comment]
Hey James,

Thank you very much for this, what you say makes perfect sense. I've added the following explanation to page 46 (I agree that's the best spot):

> You might also note that the "Bands" navigation item is marked as active. This is
part of the Ember magic (in the good sense of the word, I think). Ember marks
all `link-to` that have a destination route which is the parent of the current
route, or the exact same one (as in this case).

> It does this by adding the `active` CSS class to the `a` tag so all we had to do
was to define a CSS rule, which we did at the beginning of the chapter.

Does this work?

Extra special thanks for going beyond reporting the "bug" by suggesting where the best place to fix this would be :clap:
